src/EditorFeatures/VisualBasicTest/UseSystemHashCode/UseSimpleUsingStatementTests.vb

```diff
@@ -0,0 +1,627 @@
+﻿' Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+Imports Microsoft.CodeAnalysis.CodeFixes
+Imports Microsoft.CodeAnalysis.Diagnostics
+Imports Microsoft.CodeAnalysis.Editor.VisualBasic.UnitTests.Diagnostics
+Imports Microsoft.CodeAnalysis.UseSystemHashCode
+
+Namespace Microsoft.CodeAnalysis.Editor.VisualBasic.UnitTests.UseSimpleimportsStatement
+    Partial Public Class UseSystemHashCodeTests
```

Comment:
File doesn't match class name -- I'm guessing you copied from something else and didn't update.

Reply:
Dne.